interceptors/interceptors.go

```diff
@@ -43,10 +43,10 @@ func DefaultInterceptors() []grpc.UnaryServerInterceptor {
 func DefaultClientInterceptors(address string) []grpc.UnaryClientInterceptor {
 	return []grpc.UnaryClientInterceptor{
 		grpc_retry.UnaryClientInterceptor(),
-		GRPCClientInterceptor(),
 		NewRelicClientInterceptor(address),
 		HystrixClientInterceptor(),
 		ForwardMetadataInterceptor(),
+		GRPCClientInterceptor(),
```

Comment:
Order of `GRPCClientInterceptor`  `ForwardMetadataInterceptor` important.
Because the trace headers propagated to the caller service should be of the current span, not the upstream span present in the incoming metadata.